cni/pkg/ambient/server.go

```diff
@@ -152,25 +151,26 @@ func (s *Server) UpdateConfig() {
 
 var ztunnelLabels = labels.ValidatedSetSelector(labels.Set{"app": "ztunnel"})
 
-func (s *Server) ReconcileZtunnel() error {
+func (s *Server) UpdateActiveNodeProxy() error {
 	pods := s.pods.List(metav1.NamespaceAll, ztunnelLabels)
 	var activePod *corev1.Pod
 	for _, p := range pods {
+		log.Debugf("Examining ztunnel pod - %s", p.Name)
```

Comment:
nit: at top of function can set `log := log.WithScope("pod", p.Name)` to avoid repetition and have more structured logs